FREQUENCY: 10 #Hz.
LINEAR_VELOCITY: 0.2 # m/s
ANGULAR_VELOCITY: math.pi/4 # rad/s
DEFAULT_CMD_VEL_TOPIC: 'cmd_vel'
DEFAULT_ERROR_TOPIC: 'error'